pkg/background/generate/generate.go

```diff
@@ -704,6 +708,15 @@ func manageCloneList(log logr.Logger, namespace, policy string, clone kyvernov1.
 			// check if the resource as reference in clone exists?
 			obj, err := client.GetResource(apiVersion, kind, rNamespace, rName.GetName())
 			if err != nil {
+				if apierrors.IsNotFound(err) && len(ur.Status.GeneratedResources) != 0 && !synchronize {
+					log.V(4).Info("synchronize is disable - skip re-create", "resource", obj)
```

Comment:
"synchronization is disabled, recreation will be skipped"

Reply:
fixed